operator-common/src/main/java/io/strimzi/operator/common/Util.java

```diff
@@ -539,4 +546,56 @@ public static String fromAsciiBytes(byte[] bytes) {
         return new String(bytes, StandardCharsets.US_ASCII);
     }
 
+    /**
+     * Build basic HTTP authentication header value.
+     * 
+     * @param username Username.
+     * @param password Password.
+     * 
+     * @return Header value.
+     */
+    public static String buildBasicAuthValue(String username, String password) {
+        String credentials = join(":", username, password);
+        return format("Basic %s", Util.encodeToBase64(credentials));
+    }
+
+    /**
+     * Get file content as string.
+     * 
+     * @param filePath File path.
+     * 
+     * @return Content as string.
+     */
+    public static String stringFromFile(File filePath) {
```

Comment:
Seems to be used only in TO tests. So it should live in TO tests.

Reply:
Ok, moved to TO util. Thanks.